### PR TITLE
Game bss and cleanup

### PR DESCRIPTION
**Pull request by hensldm**
* Imports the bss for Game as well as some general clean up.
* Creates a header for speedmeter.

If anyone has naming suggestions for `func_80173B48` I'm all ears.

### CODE REVIEW

File: include/z64speed_meter.h [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#ifndef Z64SPEED_METER_H
+#define Z64SPEED_METER_H
+
+#include "PR/ultratypes.h"
+
+typedef struct SpeedMeter {
+    /* 0x00 */ char unk_00[0x1C];
+} SpeedMeter; // size = 0x20
```
Comment (AngheloAlf) [inline include/z64speed_meter.h:8]: The size of the struct and the size of the comment doesn't match

File: include/z64speed_meter.h [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+struct GameState;
+struct GraphicsContext;
```
Comment (AngheloAlf) [inline include/z64speed_meter.h:11]: iirc we have been putting those struct forward declaration at the top of the file, before other struct definitions. Could you move them up?

**Comment by hensldm** [comment]
Small note changed the naming of the RDP time variables to match OoT

**Review by AngheloAlf — APPROVED** [review]

File: src/code/z_kaleido_setup.c [hunk 3]
```diff
@@ -110,7 +110,7 @@ void KaleidoSetup_Update(PlayState* play) {
                                 }
 
                                 if (pauseCtx->state == PAUSE_STATE_OPENING_0) {
-                                    Game_SetFramerateDivisor(&play->state, 2);
+                                    GameState_SetFramerateDivisor(&play->state, 2);
```
Comment (engineer124) [inline src/code/z_kaleido_setup.c:113]: This can probably be thrown in `namefixer.py`

**Review by engineer124 — APPROVED** [review]

**Review by jpburnett — APPROVED** [review]

**Pull request merged**